pkg/utils/schema.go

```diff
@@ -99,26 +109,22 @@ func LoadBackupTables(meta *backup.BackupMeta) (map[string]*Database, error) {
 				return nil, errors.Trace(err)
 			}
 		}
-		partitions := make(map[int64]bool)
+		partitions := make(map[int64]struct{})
 		if tableInfo.Partition != nil {
 			for _, p := range tableInfo.Partition.Definitions {
-				partitions[p.ID] = true
+				partitions[p.ID] = struct{}{}
 			}
 		}
 		// Find the files belong to the table
-		tableFiles := make([]*backup.File, 0)
-		for _, file := range meta.Files {
-			// If the file do not contains any table data, skip it.
-			if !bytes.HasPrefix(file.GetStartKey(), tablecodec.TablePrefix()) &&
-				!bytes.HasPrefix(file.GetEndKey(), tablecodec.TablePrefix()) {
-				continue
-			}
```

Comment:
seems this behaviour is replaced to panic 🤔 Is this OK?

Reply:
Table ID must never be 0. There must be something wrong if happens.